purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -2010,6 +2014,13 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             features: List<BillingFeature> = listOf(),
             callback: Callback<Boolean>
         ) {
+            if (!checkMethodAvailability(
+                    listOf(Store.PLAY_STORE),
+                    "canMakePayments",
+                    "returning false")) {
+                callback.onReceived(false)
```

Comment:
i wonder if defaulting to true is safer, in case a user has existing logic to avoid purchase flows if this returns false? @vegaro do you have any context on amazon's "billing availability"?

Reply:
I think we should return true for those reasons. there's no Amazon way of checking if it is available. If it ends up not being available, there will be an error at some point (probably thrown by the Amazon store itself), so I think it's better to be safe and return true.